### ISSUE TITLE
Run test to detect schema migration failures

### ISSUE THREAD

**Issue by PabloCastellano** [Blobscan/blobscan#109]
For example, migration `20230716183410_add_inserted_at_and_upserted_at_fields` could not be applied in production because insertedAt is a new non-nullable field without a default value.

```
A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20230716183410_add_inserted_at_and_upserted_at_fields

Database error code: 23502

Database error:
ERROR: column "insertedAt" of relation "Address" contains null values

DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E23502), message: "column \\"insertedAt\\" of relation \\"Address\\" contains null values", detail: None, hint: None, position: None, where_: None, schema: Some("public"), table: Some("Address"), column: Some("insertedAt"), datatype: None, constraint: None, file: Some("tablecmds.c"), line: Some(5896), routine: Some("ATRewriteTable") }

   0: sql_schema_connector::apply_migration::apply_script
           with migration_name="20230716183410_add_inserted_at_and_upserted_at_fields"
             at schema-engine/connectors/sql-schema-connector/src/apply_migration.rs:106
   1: schema_core::commands::apply_migrations::Applying migration
           with migration_name="20230716183410_add_inserted_at_and_upserted_at_fields"
             at schema-engine/core/src/commands/apply_migrations.rs:91
   2: schema_core::state::ApplyMigrations
             at schema-engine/core/src/state.rs:197
```

**Comment by PabloCastellano** [comment]
Fixed in Vercel itself (commit b44c31a56ad9475d469ca40b94fe0cb5446cc3b0)

**Comment by PabloCastellano** [comment]
Fixed in Vercel itself (commit https://github.com/Blobscan/blobscan/commit/b44c31a56ad9475d469ca40b94fe0cb5446cc3b0)

![image](https://github.com/Blobscan/blobscan/assets/73274/c0fe2e3f-fe82-45e6-89b9-6b8f16cd7a08)
